i18n/fr/docusaurus-plugin-content-docs/current/contribution.md

```diff
@@ -29,6 +29,8 @@ We want to thank:<br/>
 @386861188891279362 - Website Architect / Documentation Organizer<br/>
 @102102717165506560 - Growth Consulting / Resource Curator<br/>
 @755792681313108018 - *I had this idea and did all of the above*
+@480495309491798037 - http://discord.wiki/ Redirect Owner <br/>
```

Comment:
the wording here is a bit confusing if you don't know what it means already
suggestion: owner of the "discord.wiki" domain alias